src/App.tsx

```diff
@@ -1,24 +1,254 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useRef, useState } from 'react';
+import { ErrorType, Todo } from './types/Todo';
+import {
+  getTodos,
+  postTodo,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
+import { ErrorPopup } from './components/ErrorPopup';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
 
-const USER_ID = 0;
+const USER_ID = 11546;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filterType, setFilterType] = useState<'All' | 'Active' | 'Completed'>(
```

Comment:
Rember about the DRY rule.
Let's export this to type defintion and use it in all 5 places,